conda_build/source.py

```diff
@@ -69,25 +72,34 @@ def download_to_cache(meta):
     return path
 
 
-def unpack(meta):
+def unpack(meta, verbose=False):
     ''' Uncompress a downloaded source. '''
     src_path = download_to_cache(meta)
 
     os.makedirs(WORK_DIR)
-    print("Extracting download")
+    if verbose:
+        print("Extracting download")
     if src_path.lower().endswith(('.tar.gz', '.tar.bz2', '.tgz', '.tar.xz',
             '.tar', 'tar.z')):
         tar_xf(src_path, WORK_DIR)
     elif src_path.lower().endswith('.zip'):
         unzip(src_path, WORK_DIR)
     else:
         # In this case, the build script will need to deal with unpacking the source
-        print("Warning: Unrecognized source format. Source file will be copied to the SRC_DIR")
+        if verbose:
+            print("Warning: Unrecognized source format. Source file will be copied to the SRC_DIR")
```

Comment:
Should this warning be raised even if `verbose` is `False`? I normally think of verbose as "no chatter, but if something goes wrong please do tell me."